static/app/components/stories/sizingWindow.tsx

```diff
@@ -26,4 +27,5 @@ const SizingWindow = styled(NegativeSpaceContainer)<Props>`
       `}
 `;
 
+export type Props = ComponentProps<typeof SizingWindow>;
```

Comment:
avoid exporting anything called props. is this used?

Reply:
Good catch! Just moved it and did not pay attention. It is used only in https://github.com/getsentry/sentry/blob/master/static/app/components/stories/matrix.tsx
Will rename 👍